src/Analyzers/CSharp/Analyzers/UsePatternMatching/CSharpUseNotPatternDiagnosticAnalyzer.cs

```diff
@@ -64,6 +64,7 @@ private void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext)
 
             // Look for the form: !(x is Y y)
             if (!(node is PrefixUnaryExpressionSyntax
+#pragma warning disable format
```

Comment:
File a tracking issue to revert this PR when we upgrade CodeStyle layer that matches built-in analyzer behavior?